core-tests/src/test/java/org/mule/runtime/core/DefaultMessageContextTestCase.java

```diff
@@ -51,7 +51,7 @@ public void before() {
 
   @Test
   public void noCorrelationIdInContext() {
-    final MessageContext executionContext = create(flow);
+    final MessageContext executionContext = create(flow, "test");
```

Comment:
- Refactor to use a test utility method rather than passing string "test" each time ideally.  Even if there isn't a util method this should be a constant defined once and reused.
- Static import is not ideal with `create`